flink-metrics/flink-metrics-datadog/src/main/java/org/apache/flink/metrics/datadog/DatadogHttpReporter.java

```diff
@@ -43,6 +45,8 @@
  *
  * <p>Variables in metrics scope will be sent to Datadog as tags.
  */
+@PublicEvolving
```

Comment:
same issue a https://github.com/apache/flink/pull/11778#discussion_r409491135.

Reply:
this issue has been handled and submitted, please check it again,thx